python/cudf/cudf/core/column/string.py

```diff
@@ -3900,6 +3903,92 @@ def replace_tokens(self, targets, replacements, delimiter=None, **kwargs):
+        hash_file : str
+            Path to hash file containing vocabulary of words with token-ids.
+        max_length : int, Default is 64
+            Limits the length of the sequence returned.
+            If tokenized string is shorter than max_length,
+            output will be padded with 0s.
+            If the tokenized string is longer than max_length and
+            do_truncate == False, there will be multiple returned
+            sequences containing the overflowing token-ids.
+        stride : int, Default is 48
+            If do_truncate == False and the tokenized string is larger
+            than max_length, the sequences containing the overflowing
+            token-ids can contain duplicated token-ids from the main
+            sequence. If max_length is equal to stride there are no
+            duplicated-id tokens. If stride is 80% of max_length,
+            20% of the first sequence will be repeated on the second
+            sequence and so on until the entire sentence is encoded.
+        do_lower : bool, Default is True
+            If set to true, original text will be lowercased before encoding.
+        do_truncate : bool, Default is False
+            If set to true, strings will be truncated and padded to
+            max_length. Each input string will result in exactly one output
+            sequence. If set to false, there may be multiple output
+            sequences when the max_length is smaller than generated tokens.
+        max_num_strings : int, Default is 100
+            The maximum number of strings to be encoded.
+        max_num_chars : int, Default is 100000
+            The maximum number of characters in the input strings column.
+        max_rows_tensor : int, Default is 500
+            The maximum number of rows in the output
+
+        Returns
+        -------
+        token-ids : Column
+            The token-ids for each string padded with 0s to max_length.
+        attention-mask : Column
+            The mask for token-ids result where corresponding positions
+            identify valid token-id values.
+        metadata : Column
+            Each row contains the index id of the original string and the
+            first and last index of the token-ids that are non-padded and
+            non-overlapping.
+
+        Examples
+        --------
+        >>> import cudf
+        >>> ser = cudf.Series(['this is the', 'best book'])
+        >>> tokens, masks, metadata =
+               ser.str.subword_tokenize("bert_hash_table.txt")
+        """
+        tokens, masks, metadata = cpp_subword_tokenize(
+            self._column,
+            hash_file,
+            max_length,
+            stride,
+            do_lower,
+            do_truncate,
+            max_num_strings,
+            max_num_chars,
+            max_rows_tensor,
+        )
+        return (
+            cudf.Series(tokens),
+            cudf.Series(masks),
+            cudf.Series(metadata),
+        )
```

Comment:
I'm thinking these should be cupy arrays instead of `cudf.Series`. Using a Series here implies that there's useful indexing information, which isn't true and Column objects aren't user facing as of now.

Reply:
```suggestion
        return (
            cupy.asarray(tokens),
            cupy.asarray(masks),
            cupy.asarray(metadata),
        )
```